### PR TITLE
feat: Add Readwise Reader import

### PR DESCRIPTION
**Pull request by cinconueves**
As the title suggests: This PR adds the functionality of importing from Readwise Reader's CSV export file. 

image

See issue #2588 for more information.

### CODE REVIEW

File: packages/shared/import-export/parsers.ts [hunk 1]
```diff
@@ -457,6 +458,82 @@ function parseInstapaperBookmarkFile(textContent: string): ParsedBookmark[] {
...
+  const record = parse(textContent, {
+    columns: true,
+    skip_empty_lines: true,
+    cast: function (value, context) {
+      //Replace for json parsing; original comes with \' instead of \" so it's not json parsable.
+      if (context.index === 3) {
+        return value.replace(/'/g, '"');
+      }
+      return value;
+    },
```
[the remark this answers is not in the document]
Comment (cinconueves) [reply packages/shared/import-export/parsers.ts:526]: Fixed

File: packages/shared/import-export/parsers.ts [hunk 2]
```diff
@@ -457,6 +458,82 @@ function parseInstapaperBookmarkFile(textContent: string): ParsedBookmark[] {
...
+    try {
+      const documentTags = record["Document tags"];
+      if (documentTags) {
+        const parsedTags = JSON.parse(documentTags);
+        if (Array.isArray(parsedTags)) {
+          tags = parsedTags.map((tag) => tag.toString().trim());
+        }
+      }
+    } catch {
+      tags = [];
```
[the remark this answers is not in the document]
Comment (cinconueves) [reply packages/shared/import-export/parsers.ts:563]: Fixed, added tests and handled edge cases.

**Comment by cinconueves** [comment]
- Added tests to handle CSV tag edge cases & 
- Changed code to handle other scenarios

**Comment by MohamedBassem** [comment]
Thank you!
